draft-ietf-jsonpath-base.md

```diff
@@ -176,7 +176,7 @@ unchanged.
 Importantly "object" and "array" in particular do not take on a
 generic meaning, such as they would in a general programming context.
 
-Additional terms used in this specification are defined below.
+Additional terms used in this document are defined below.
```

Comment:
I'm mostly neutral to this change.
(In this particular place, it is an actual improvement, but not necessarily elsewhere.)
"This document" is more confusing in a spec that talks about documents; fortunately, we don't.

Reply:
I'm not entirely sold on it either. I'll let this PR go cold and then go through the cases and see which I think are improvements.

Reply:
RFC 8259 also uses "document" for self reference. I am ok with this change.